### DISCUSSION TITLE
Bring the old feed back

### DISCUSSION THREAD

**Discussion by dop251** [community/community#66158] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The new home feed is completely useless, please bring the old one back. In particular I want to only see activities on my repositories, which were present in the old feed, but not in the new one, even with all the filters disabled.

**Comment by mxmlnkn** [comment]
Exactly this. There are so many options, why isn't this one of them?! I'm not interested what other people are doing, it feels like stalking them. And I'm not interested in every release or discussion of projects I only starred instead of watching it.

**Reply by mxmlnkn** [reply]
In the meantime, I got pointed to https://github.com/Gerrit0/old-github-feed which works fine for me!

**Comment by wangrongding** [comment]
# Github Old Feed  

💪 Using this script you can restore the old version of the feed.
https://github.com/wangrongding/github-old-feed  

<table>
  <tr>
      <td width="50%" align="center"><b>Before</b></td>
      <td width="50%" align="center"><b>After use</b></td>
  </tr>
  <tr>
     <td><img src="https://assets.fedtop.com/picbed/1694066442147.jpg"/></td>
     <td><img src="https://assets.fedtop.com/picbed/1694066437621.jpg"/></td>
  </tr>
</table>
